.circleci/continue-workflows.yml

```diff
@@ -216,7 +216,7 @@ jobs:
         type: string
         description: working directory
     executor: go-cimg-executor
-    resource_class: large
+    resource_class: xlarge
```

Comment:
Intended?

Reply:
Yes - without this I noticed the coverage job die after using all 8GB of memory (actual usage was 7.98GB). Now we are using around 10GB of memory. We could probably either bump up the proc count or decrease the proc count and resource class instead.